_scripts/firestore.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -ex
+
+# This docker image doesn't react to SIGINTs (Ctrl+C) which is used by
+# pm2 to kill processes.
+# We go around this by defining a SIGINT handler, running docker in the
+# background (but still logging on the screen), and running a read to keep
+# the script running.
+
+function on_singint() {
```

Comment:
Typo in the function name here, although a song about ctrl+c sounds pretty cool too. 😄

Reply:
That's what I get for copy/pasting from mysql.sh. I'll fix them all.